pkg/kubelet/config/config.go

```diff
@@ -456,6 +456,7 @@ func checkAndUpdatePod(existing, ref *v1.Pod) (needUpdate, needReconcile, needGr
 	existing.Labels = ref.Labels
 	existing.DeletionTimestamp = ref.DeletionTimestamp
 	existing.DeletionGracePeriodSeconds = ref.DeletionGracePeriodSeconds
+	existing.Generation = ref.Generation
```

Comment:
check above in the `if !reflect.DeepEqual(existing.Status, ref.Status)` case specifically, to see if we hit that and hotloop if the API server has PodObservedGenerationTracking disabled and the kubelet has it enabled

Reply:
let me know if the answer in https://github.com/kubernetes/kubernetes/pull/130352#discussion_r1977730000 resolves the concern here

Reply:
um... actually... not calling dropDisabledFields from status strategy feels like a bug / oversight we should fix, which means this question is still a question